.github/actions/deploy-ubuntu/action.yml

```diff
@@ -153,6 +155,19 @@ runs:
           $SUDO mv /usr/include/cudnn* /usr/include/nccl* /usr/local/cuda/include/
           $SUDO mv /usr/lib64/libcudnn* /usr/lib64/libnccl* /usr/local/cuda/lib64/
 
+          if [[ -n ${NVCOMP:-} ]]; then
+            curl -LO https://developer.download.nvidia.com/compute/nvcomp/3.0.3/local_installers/$NVCOMP.tgz
```

Comment:
nvCOMP 3.0.4 is available for Linux as well

Reply:
Done